iot_verifier/src/hex_density.rs

```diff
@@ -193,8 +207,8 @@ fn limit(res: u8, occupied_count: u64) -> u64 {
 pub fn compute_hex_density_map(global_map: &GlobalHexMap) -> HashMap<u64, Decimal> {
     let mut map: HashMap<u64, Decimal> = HashMap::new();
     for hex in &global_map.asserted_hexes {
-        let scale: Decimal = SCALING_RES.rev().fold(dec!(1.0), |scale, res| {
-            hex.get_parent(res).map_or(scale, |parent| {
+        let scale: Decimal = SCALING_RES.iter().fold(dec!(1.0), |scale, res| {
```

Comment:
Just confirming that the previous version was going from 4..13 and now it's 13..4 (so no need to call `.rev()`)

Reply:
that's correct; these two constants used to be a range struct that incremented positively so we needed to reverse the iterator when working through it but now just creating the resolution slice in descending order